Source/Pokens.Pokedex/Pokens.Pokedex.Business/Pokemon/Models/PokemonModel.cs

```diff
@@ -23,6 +24,8 @@ public PokemonModel(Pokemon pokemon)
 
         public bool IsStarter { get; set; }
 
+        public IEnumerable<Image> Images { get; set; }
```

Comment:
Entitatile de domeniu n-ar trebui expuse spre exterior. In schimb, definim cate un Model pentru fiecare concept care ajunge in exterior (la clienti, aka aplicatia de angular)